climada/engine/impact.py

```diff
@@ -1146,42 +1164,32 @@ def plot_rp_imp(
         imp_stats : np.array
             return_periods.size x num_centroids
         """
-        imp_stats = (
-            self.local_exceedance_impact(np.array(return_periods))[0].values[:, 1:].T
+
+        LOGGER.info(
+            "Some errors in the previous calculation of local exceedance impacts have been corrected,"
+            " see Impact.local_exceedance_impact. To reproduce data with the "
+            "previous calculation, use CLIMADA v5.0.0 or less."
```

Comment:
Cool idea, I am not sure though that this should be an info logging. @emanuel-schmid : what would be the most elegant way to inform the users that something changed, without having logs all the time?